SPECS/ceph/ceph.spec

```diff
@@ -847,6 +851,7 @@ ${CMAKE} .. \
     -DWITH_MANPAGE=ON \
     -DWITH_PYTHON3=%{python3_version} \
     -DWITH_MGR_DASHBOARD_FRONTEND=OFF \
+    -DWITH_SEASTAR=OFF \
```

Comment:
Could you add a comment explaining why we disabled it, so that we don't accidentally enable it in the future without checking if the we're good CVE-wise?

Reply:
Otherwise looks good to me.

Reply:
Sounds good, added a comment about it!